speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp

```diff
@@ -48,15 +52,17 @@ Recorder::~Recorder()
 /// Only writes to the file if the input is different then the previous input.
 /// @preconditions p_input must be PARAMETERS long (in this case 4)
 /// @param p_input array of size PARAMETERS
```

Comment:
Also please edit the start of the comment `float*` to `array`

```suggestion
/// @preconditions p_input must be m_paramAmount long (in this case 4)
/// @param p_input array of size m_paramAmount 
```